dev-perl/OpenOffice-OODoc/OpenOffice-OODoc-2.125.0.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DIST_AUTHOR="JMGDOC"
+DIST_VERSION="2.125"
+
+inherit perl-module
+
+DESCRIPTION="The Perl Open OpenDocument Connector"
+
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND=">=dev-perl/XML-Twig-3.520.0
+	>=dev-perl/Archive-Zip-1.590.0"
```

Comment:
really DEPEND ?

Reply:
missed it on my review (tried to do too many in a sitting), will fix and slow down

Reply:
Out of curiosity. why is DEPEND still supported on recent EAPIs?